docs/detections/api/exceptions/api-import-exception-list.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[exceptions-api-import-exception-list]]
+=== Import exception list
+
+Imports an exception list and associated items.
+
+An exception list groups <<exceptions-api-create-exception-item, exception items>>
+and can be associated with rules. When an exception item's query evaluates to
+`true`, rules do *not* issue alerts even when the rule's other criteria are met.
+
+You can assign detection rules with multiple exception lists. For more information, refer to <<rules-api-create>> and <<rules-api-update>>.
+
+IMPORTANT: All exception items added to the same list are evaluated using
+`OR` logic. That is, if any of the items in a list evaluate to `true`, the
+exception prevents the rule from generating an alert. Likewise, `OR` logic is
```

Comment:
```suggestion
prevents the rule from generating an alert. Likewise, `OR` logic is
```

Reply:
why are we removing `exception` from this sentence? Wouldn't this read as

`That is, if any of the items in a list evaluate to `true`, the prevents the rule from generating an alert. Likewise, `OR` logic is`?

Reply:
@dhurley14 Removing `exception` works together with the edits in the line above (from `the` to `this`), so the full sentence would be:

> This means that if any of the items in a list evaluate to `true`, this prevents the rule from generating an alert.